application/models/item.go

```diff
@@ -711,13 +711,13 @@ func (i *Item) getInitialCoverage(tx *pop.Connection, now time.Time) CoveragePer
 
 	i.LoadCategory(tx, false)
 	if i.Category.GetBillingPeriod() == domain.BillingPeriodMonthly {
-		coverage.Premium = i.CalculateMonthlyPremium(tx)
 		if now.Day() < MonthlyCutoffDay {
-			coverage.EndDate = domain.EndOfMonth(now)
+			coverage.Premium = i.CalculateMonthlyPremium(tx)
 		} else {
-			oneMonthAhead := now.AddDate(0, 1, 0)
-			coverage.EndDate = domain.EndOfMonth(oneMonthAhead)
+			// After the cutoff day, no premiums are billed until the next month. See CVR-730.
+			coverage.Premium = 0
 		}
+		coverage.EndDate = domain.EndOfMonth(now)
```

Comment:
1. The default for coverage.Premium is 0, so don't need to actually set that if desired (leaving it in suggestion though).
2. I assume you probably asked, but just to be sure, should it be `now.Day() < MonthlyCutoffDay` or `now.Day() <= MonthlyCutoffDay`?

```suggestion
		// After the cutoff day, no premiums are billed until the next month. See CVR-730.
		coverage.Premium = 0
		coverage.EndDate = domain.EndOfMonth(now)

		if now.Day() < MonthlyCutoffDay {
			coverage.Premium = i.CalculateMonthlyPremium(tx)
		}
```

Or without the Premium being explicitly set (using default of 0)

```suggestion
		coverage.EndDate = domain.EndOfMonth(now)

		// After the cutoff day, no premiums are billed until the next month. See CVR-730.
		if now.Day() < MonthlyCutoffDay {
			coverage.Premium = i.CalculateMonthlyPremium(tx)
		}
```